<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CXPropertySettingDialog</class>
 <widget class="QDialog" name="CXPropertySettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>353</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>362</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>轴可见</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QCheckBox" name="timeAxisLineVisibleCheckBox">
         <property name="text">
          <string>显示</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>时间长度</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="timeFontColorButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="timeFontButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>高度</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>时间格式</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="timeAxisVisibleCheckBox">
         <property name="text">
          <string>显示</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="2">
        <widget class="QLineEdit" name="timeDayLineEdit"/>
       </item>
       <item row="12" column="6">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>小时</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="6">
        <widget class="QTextEdit" name="timeCommentTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>单位</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>说明</string>
         </property>
        </widget>
       </item>
       <item row="12" column="4" colspan="2">
        <widget class="QLineEdit" name="timeHourLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="5" colspan="2">
        <widget class="QPushButton" name="axisLineSettingButton">
         <property name="text">
          <string>轴线设置</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>字体</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="7">
        <widget class="QLineEdit" name="timeFormatLineEdit"/>
       </item>
       <item row="7" column="1" colspan="6">
        <widget class="QLineEdit" name="timeHeightLineEdit">
         <property name="toolTip">
          <string>有效值为 1-1000的整数</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="3">
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>例如: yyyy-MM-dd hh:mm:ss
        (年-月-日 时:分:秒)</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="6">
        <widget class="QLineEdit" name="timeUnitLineEdit"/>
       </item>
       <item row="12" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>天</string>
         </property>
        </widget>
       </item>
       <item row="13" column="4" colspan="2">
        <widget class="QLineEdit" name="timeSecondLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="2">
        <widget class="QLineEdit" name="timeMinuteLineEdit"/>
       </item>
       <item row="13" column="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>分</string>
         </property>
        </widget>
       </item>
       <item row="13" column="6">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>秒</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4" colspan="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>113</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>轴线</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="3">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>轴线</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="4">
        <widget class="QTextEdit" name="valueCommentTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>轴可见</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>最小值</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="valueAxisVisibleCheckBox">
         <property name="text">
          <string>显示</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2" colspan="4">
        <widget class="QLineEdit" name="minValueLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>高度</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="4">
        <widget class="QLineEdit" name="maxValueLineEdit">
         <property name="toolTip">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>描述</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="valueFontButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>最大值</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>单位</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2" colspan="4">
        <widget class="QLineEdit" name="valuePrecisionLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>有效值为1~5的整数</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="4">
        <widget class="QLineEdit" name="valueUnitLineEdit"/>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>精度</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="valueFontColorButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="4">
        <widget class="QLineEdit" name="valueHeightLineEdit">
         <property name="toolTip">
          <string>有效值为 1-1000的整数</string>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>字体</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QCheckBox" name="valueAxisLineVisibleCheckBox">
         <property name="text">
          <string>显示</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="valueAxisLineSettingButton">
         <property name="text">
          <string>轴线设置</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>113</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="2">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="axisTypeComboBox">
       <item>
        <property name="text">
         <string>时间</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>数值</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>类型</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>名称</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>axisTypeComboBox</tabstop>
  <tabstop>timeAxisVisibleCheckBox</tabstop>
  <tabstop>timeFontButton</tabstop>
  <tabstop>timeCommentTextEdit</tabstop>
  <tabstop>timeUnitLineEdit</tabstop>
  <tabstop>timeHeightLineEdit</tabstop>
  <tabstop>timeFormatLineEdit</tabstop>
  <tabstop>timeDayLineEdit</tabstop>
  <tabstop>timeHourLineEdit</tabstop>
  <tabstop>timeMinuteLineEdit</tabstop>
  <tabstop>timeSecondLineEdit</tabstop>
  <tabstop>valueAxisVisibleCheckBox</tabstop>
  <tabstop>valueFontButton</tabstop>
  <tabstop>valueCommentTextEdit</tabstop>
  <tabstop>valueHeightLineEdit</tabstop>
  <tabstop>maxValueLineEdit</tabstop>
  <tabstop>minValueLineEdit</tabstop>
  <tabstop>valuePrecisionLineEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>valueUnitLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
